packages/prerender/package.json

```diff
@@ -10,8 +10,8 @@
   ],
   "main": "dist/index.js",
   "peerDependencies": {
-    "react": "*",
-    "react-dom": "*"
+    "react": "17.0.2",
+    "react-dom": "17.0.2"
```

Comment:
@dac09 the `*` came up in a Yarn check as a warning. Ok to do this here or should I revert?

Reply:
Totally OK to do, but I think `*` would be better, since its a peer dependency.

Maybe yarn 3 has a new way of setting it